Documentation/dev-guide/api_reference_v3.md

```diff
@@ -616,6 +616,7 @@ Empty field.
 | value | value is the value, in bytes, to associate with the key in the key-value store. | bytes |
 | lease | lease is the lease ID to associate with the key in the key-value store. A lease value of 0 indicates no lease. | int64 |
 | prev_kv | If prev_kv is set, etcd gets the previous key-value pair before changing it. The previous key-value pair will be returned in the put response. | bool |
+| no_value | If no_value is set, etcd updates the key using its current value. Returns an error if the key does not exist. | bool |
```

Comment:
@heyitsanthony yes i understand your point of view. But my point of view was it may seem confusing about storing only key, since put is applicable for both new and old values and here only update is allowed.